packages/entity/src/ComposedEntityCacheAdapter.ts

```diff
@@ -40,11 +44,35 @@ export default class ComposedEntityCacheAdapter<TFields> extends EntityCacheAdap
           newUnfulfilledFieldValues.push(fieldValue);
         } else {
           retMap.set(fieldValue, cacheResult);
+          const fulfilledFieldValues = nullthrows(fulfilledFieldValuesByCacheIndex[i]);
+          fulfilledFieldValues.push(fieldValue);
```

Comment:
```suggestion
          nullthrows(fulfilledFieldValuesByCacheIndex[i]).push(fieldValue);
```
(makes it clearer that this is a object reference rather than a local variable)